src/web/builds.rs

```diff
@@ -286,25 +325,30 @@ mod tests {
                 .create()?;
 
             let response = env.frontend().get("/crate/foo/0.1.0/builds").send()?;
-            assert_cache_control(&response, CachePolicy::NoCaching, &env.config());
-            let page = kuchikiki::parse_html().one(response.text()?);
 
-            let rows: Vec<_> = page
-                .select("ul > li a.release")
-                .unwrap()
-                .map(|row| row.text_contents())
-                .collect();
+            // FIXME: temporarily we don't show in-progress releases anywhere, which means we don't
```

Comment:
Should we instead show a page saying "in the build queue" or something like that? If so, please open an issue and assign it to me. :)

Reply:
@GuillaumeGomez 
In my mind we would show them on the build queue page on the top, though that TBD. 

I found that I wanted to reduce the amount of testing / amount of changes for this PR, and then look at in progress builds separately.